service/initialise.go

```diff
@@ -94,7 +94,19 @@ func (e *ExternalServiceList) GetKafkaConsumer(ctx context.Context, cfg *config.
 // DoGetKafkaConsumer returns a Kafka Consumer group
 func (e *Init) DoGetKafkaConsumer(ctx context.Context, cfg *config.Config) (dpkafka.IConsumerGroup, error) {
 	cgChannels := dpkafka.CreateConsumerGroupChannels(1)
-	kafkaOffset := dpkafka.OffsetNewest
+
+	pConfig := &dpkafka.ProducerConfig{
```

Comment:
this is a producer config, and `pConfig` is not used in this function
I think you need `ConsumerGroupConfig` here (perhaps called `cConfig`) and to use that in the below